k8sfv/metric.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"flag"
+	"net/http"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func getFelixMetric(name string) (metric string) {
+	resp, err := http.Get("http://" + flag.Arg(1) + ":9091/metrics")
```

Comment:
It's provided by the Makefile on the ginkgo command line.

I've used a BeforeSuite to refactor and make this much clearer (I hope).